### DISCUSSION TITLE
Issue with ZOWE and IBM Z Open Editor

### DISCUSSION THREAD

**Discussion by TerminalGambit** [IBM/zopeneditor-about#339] [Q&A]
Hello everyone, 

I started a course on Coursera provided by IBM on an introduction to COBOL language. As a started lab 2, I tried to find what is my "https://url:port" address, I cannot find it. 

Can someone explain how to find it?

Thank you in advance,

JM

**Comment by phaumer** [comment]
Hello. The training class is not provided by the Z Open Editor team. Does Coursera have a user forum or tech support?
